ui/imports/shared/views/HistoryView.qml

```diff
@@ -136,29 +136,47 @@ ColumnLayout {
         ScrollBar.vertical: StatusScrollBar {}
 
         section.property: "date"
-        section.delegate: Item {
+        section.delegate: ColumnLayout {
             id: sectionDelegate
 
             readonly property bool isFirstSection: ListView.view.firstSection === section
 
             width: ListView.view.width
-            height: isFirstSection || section.length > 1 ? 40 : 0 // 1 because we don't use empty for loading state
+            height: isFirstSection ? 38 : (section.length > 1 ? 58 : 0) // 1 because we don't use empty for loading state
```

Comment:
Do we really need height here it this is `ColumnLayout`?

Reply:
Yes. Instead each component needs to handle it's own size so ColumnLayout will use children height.

Reply:
What I meant is, if all components placed in column layout have their implicit heights defined we wouldn't need to calculate/set the height of it manually.

Reply:
Removed height from ColumnLayout. Children now handle column height